Mathlib/Analysis/Calculus/FormalMultilinearSeries.lean

```diff
@@ -362,6 +362,17 @@ theorem constFormalMultilinearSeries_apply [NontriviallyNormedField 𝕜] [Norme
   Nat.casesOn n (fun hn => (hn rfl).elim) (fun _ _ => rfl) hn
 #align const_formal_multilinear_series_apply constFormalMultilinearSeries_apply
 
+@[simp]
```

Comment:
Can you generalise this to any `c : F` (or am I misunderstanding how `FormatMultilinearSeries` works)?

Reply:
This is saying the injection from the field commutes with Zero, so I don’t think it sensibly generalizes.